contracts/cheddar_5x5_tic_tac_toe/src/game.rs

```diff
@@ -137,25 +148,32 @@ impl Game {
         self.reward.clone()
     }
 
-    pub fn get_opponent(&self, player: &AccountId)->AccountId{
+    pub fn get_opponent(&self, player: &AccountId) -> AccountId {
         if *player == self.current_player_account_id() {
             return self.next_player_account_id();
         } else {
             return self.current_player_account_id();
         }
     }
 
-    pub fn claim_timeout_win(&self, player: &AccountId) -> bool{
+    pub fn claim_timeout_win(&self, player: &AccountId) -> bool {
```

Comment:
Entry functions should not use reference (except `self`).

```suggestion
    pub fn claim_timeout_win(&self, player: AccountId) -> bool {
```